### DISCUSSION TITLE
(v0.50.34) Not able to setup the local system for Open-source contribution

### DISCUSSION THREAD

**Discussion by vadhavana** [airbytehq/airbyte#32198] [General]
![image](https://github.com/airbytehq/airbyte/assets/32058734/dbf31af8-56e1-4736-8474-95b3fa946840)

**Comment by faelmori** [comment]
You need to change version on .env file from dev to another valid version.

**Reply by vadhavana** [reply]
How do I know the valid version?

**Reply by faelmori** [reply]
![image](https://github.com/airbytehq/airbyte/assets/80360183/e414b0dc-2130-41c2-a4fd-a41abcfdaa96)

put only numbers with dots like: 0.50.34

Hope this helps..

**Reply by vadhavana** [reply]
Thanks!

**Comment by marcosmarxm** [comment]
What command are you running to run the platform?

**Reply by vadhavana** [reply]
By running "./run-ab-platform.sh"  command

Also this is taking long to start the server.

![image](https://github.com/airbytehq/airbyte/assets/32058734/31a304ac-ddf1-4861-8097-54cc10e355fe)

![image](https://github.com/airbytehq/airbyte/assets/32058734/ce51651e-4809-48fd-89c5-bc90a2ee0284)

[a comment by an automated account is not shown]
